docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,408 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #91.        prefix = np.power(2, int(np.ceil(np.log(m) / np.log(2))))
</pre><p> </p><p>According to the paper, MASS works better if the length of time series is a power of two. Two notes here:</p><p>(1) Make sure the lenght of time series that will be used is EXACTLY a power of two, and not just close it.</p><p>(2) Is <code>MASS</code> "really" faster when the length of time series is a power of two? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
(1) is done. (2) can be investigated later if needed.